openhands/app_server/sandbox/remote_sandbox_service.py

```diff
@@ -122,18 +122,9 @@ async def _send_runtime_api_request(
             _logger.error(f'HTTP error for URL {url}: {e}')
             raise
 
-    async def _to_sandbox_info(
+    def _to_sandbox_info(
```

Comment:
Looks like this is updated consistently, but could you run a mypy report and confirm we have type coverage on the updated stuff? I can show you if you don't recall where that is.